test/Garnet.test/RespTransactionProcTests.cs

```diff
@@ -370,7 +370,7 @@ public void TransactionHashProcTest()
         [Test]
         public void TransactionProcFinalizeTest()
         {
-            server.Register.NewTransactionProc("GETTWOKEYSNOTXN", 2, () => new GetTwoKeysNoTxn());
+            server.Register.NewTransactionProc("GETTWOKEYSNOTXN", () => new GetTwoKeysNoTxn(), new RespCommandsInfo { Arity = 3 });
```

Comment:
why is Arity 1 more than the earlier number of parameters? also, can the arity be specified as negative to signify "at least"?

Reply:
resolved as discussed.